src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -219,7 +219,7 @@ def load_command_table(self, _):
 
     with self.command_group('webapp deployment source') as g:
         g.custom_command('config-local-git', 'enable_local_git')
-        g.custom_command('config-zip', 'enable_zip_deploy_webapp', exception_handler=ex_handler_factory())
+        g.custom_command('config-zip', 'enable_zip_deploy_webapp', exception_handler=ex_handler_factory(), deprecate_info=g.deprecate(redirect='az webapp deploy', hide=True))
```

Comment:
May I ask when the expected deletion time for this command is? Next breaking change window (**Build Sprint**) or the one after next one (**Ignite Sprint**)?